storage/engine/rocksdb/db.cc

```diff
@@ -1381,14 +1381,13 @@ DBStatus DBImpl::Get(DBKey key, DBString* value) {
 }
 
 DBStatus DBBatch::Get(DBKey key, DBString* value) {
-  std::string encoded_key = EncodeKey(key);
-  DBGetter base(rep, read_opts, encoded_key);
+  DBGetter base(rep, read_opts, EncodeKey(key));
```

Comment:
This part of the change is just for consistency with the other uses of DBGetter, and not a functional change, right?